mne/viz/topo.py

```diff
@@ -350,9 +351,20 @@ def _plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
 
     scalings = _handle_default('scalings', scalings)
     evoked = [e.copy() for e in evoked]
+
+    # HACK development version...
+    merge_grads = True
+
     for e in evoked:
-        for pick, t in zip(picks, types_used):
-            e.data[pick] = e.data[pick] * scalings[t]
+        for pick, ch_type in zip(picks, types_used):
+            e.data[pick] = e.data[pick] * scalings[ch_type]
+
+            if ch_type == 'grad' and merge_grads:
+                from ..channels.layout import _merge_grad_data
+
+                # HACK replace mag data with grad_norm!
+                # seems to be doing something meaningful?
+                e.data[picks[0]] = _merge_grad_data(e.data[pick])
```

Comment:
And you probably want to call ``_pair_grad_sensors`` before ``_merge_grad_data``.